### DISCUSSION TITLE
Techno-Economic Assessment of Green Hydrogen Production (Project 236)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#72] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project #236](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/tree/main/projects/Techno-Economic%20Assessment%20of%20Green%20Hydrogen%20Production)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by huntermanana** [comment]
Hi, I am a student at Makerere University and interested in working and collaborating on this project with everyone.

**Reply by universe1299** [reply]
"Hi, I am a Chemical Engineering graduate from IIT Tirupati and interested in working and collaborating on this project

**Comment by estevesnatasha** [comment]
Hi, I'm a phd fellow in electrical engineering and I've been working with hydrogen for the last 10 years. I would like to participate in this discussion

**Comment by Namusoke-percy** [comment]
Hi, i am student at Gulu university and interested in working and collaborating on this project [project #236]

**Comment by universe1299** [comment]
Hi, Thanks for your interest. I'm currently working in a organisation and
will not be able to prioritise this project at the moment.
